.github/workflows/compile.yml

```diff
@@ -38,7 +34,7 @@ jobs:
     - name: Compile All
       run: |
         ./make
-        echo "ARTIFACT_NAME=${GITHUB_WORKFLOW}-$(git rev-parse --short ${GITHUB_SHA})-${GITHUB_RUN_NUMBER}" >> "${GITHUB_ENV}"
+        echo "ARTIFACT_NAME=${GITHUB_WORKFLOW}-${GITHUB_REF_NAME}-$(git rev-parse --short ${GITHUB_SHA})-${GITHUB_RUN_NUMBER}" >> "${GITHUB_ENV}"
```

Comment:
@gretel see https://github.com/ToyKeeper/anduril/pull/6#discussion_r1382676058

Reply:
yea that's better